src/utils/api.ts

```diff
@@ -6,6 +6,8 @@ import {ObjectMap, TeslerResponse} from '../interfaces/objectMap'
 import {$do} from '../actions/actions'
 import {OperationError} from '../interfaces/operation'
 import {BusinessError, ApplicationErrorType} from '../interfaces/view'
+import {openButtonWarningNotification} from './notifications'
+import {historyObj} from '..'
```

Comment:
I think it would be better if we explicitly declare on which module we are relying here

Reply:
Ok, changed path to '../reducers/router'